src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -105,7 +106,7 @@ class Lambda {
 
   private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
     if (sendFastlyPurgeRequest(contentId, purgeType, serviceId, surrogateKey, fastlyApiKey))
-      sendAmpPingRequest(contentId)
+      AmpFlusher.sendAmpDeleteRequest(contentId)
```

Comment:
As a Java dev, the way the Guardian use static / object methods catches my eye but doesn't effect the outcome.